### ISSUE TITLE
Name idea

### ISSUE THREAD

**Issue by ERRORCODE509** [foulman/Fantasy#41]
I saw that you haven't decided on a name yet. Have you considered Cataclysm: Dark Ages Ahead?

**Comment by foulman** [comment]
You're not wrong about the name, my original intent was to put off things like that until a more firm idea of lore and story had evolved, so "Fantasy" is a placeholder.
Hopefully, I will be able to return to active development in the next few weeks, and then I will consider things of that nature.

**Comment by ERRORCODE509** [comment]
All good man, take your time.

**Comment by MNG-cataclysm** [comment]
I have some ideas for a name listed:
Azcania
Azcadia
Arcadia
Hundesonne 
Bearing Day
Long Day
Long Night
Bearing Night
Red Mourning
Red Morning

**Comment by foulman** [comment]
I guess it would really depend on what's going ON in this fantasy world, and sad to say I don't really know how things stand! 😆 
I do hope to find out soon though.
Sadly working on cata gets my anxiety up thanks to Korg's jerk behavior, and I just can't add that on top rn.
